pkg/cmd/util/output/plugin_printer.go

```diff
@@ -57,7 +69,48 @@ func sortByKindAndName(plugins []velerov1api.PluginInfo) {
 func printPlugin(plugin velerov1api.PluginInfo) []metav1.TableRow {
 	row := metav1.TableRow{}
 
-	row.Cells = append(row.Cells, plugin.Name, plugin.Kind)
+	row.Cells = append(row.Cells, plugin.Name, plugin.Kind, isDefaultPlugin(plugin.Name))
 
 	return []metav1.TableRow{row}
 }
+
+func isDefaultPlugin(name string) bool {
+	for _, p := range plugin.DefaultPlugins {
+		if strings.ToLower(p) == strings.ToLower(name) {
+			return true
+		}
+	}
+	return false
+}
+
+func printPluginEncoded(list *velerov1api.ServerStatusRequest, format string) ([]byte, error) {
+	var info []pluginInfo
+
+	for _, p := range list.Status.Plugins {
+		info = append(info, pluginInfo{
+			Name:                p.Name,
+			Kind:                p.Kind,
+			SystemDefaultPlugin: isDefaultPlugin(p.Name),
+		})
+	}
+
+	switch format {
+
+	case "yaml":
+		res, err := yaml.Marshal(info)
+		if err != nil {
+			return []byte{}, err
+		}
+		return res, err
+
+	case "json":
+		res, err := json.MarshalIndent(info, "", "\t")
+		if err != nil {
+			return []byte{}, err
+		}
+		return res, err
+
+	default:
+		return []byte{}, errors.Errorf("format %s is not supported", format)
```

Comment:
I don't know the reason _why_ we need to handle this manually.
Isn't it be covered by https://github.com/vmware-tanzu/velero/blob/4ae55bb/pkg/cmd/util/output/output.go#L105-L121 already?